src/providers/token-lists.provider.ts

```diff
@@ -34,6 +35,7 @@ export interface TokenListsProviderResponse {
 
 /** SETUP */
 const { uris } = tokenListService;
+const { NETWORK_ID } = useNetwork();
```

Comment:
We should always be accessing a reactive property for the network. e.g. networkID.